app/controllers/staff/activities_controller.rb

```diff
@@ -16,10 +16,10 @@ def show
   end
 
   def create
-    @activity = policy_scope(Activity).new
+    @activity = Activity.new
+    @activity.hierarchy = hierarchy
```

Comment:
Assigning this before we call `authorize` method in all controllers allows Pundit to use this state inside it's authorisation methods, eg. `record.hierarchy` will now return the hierarchy. The other way around it will return `nil`.